kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/RecordConverter.java

```diff
@@ -29,5 +29,6 @@
    * @param record The record to convert.
    * @return The converted record.
    */
-  R convertRecord(SinkRecord record);
+  R convertRecord(SinkRecord record, boolean convertKey);
```

Comment:
Same as above
nit: maybe cleaner to have 2 functions `convertKeyRecord()` & `convertValueRecord()`?

Reply:
Updated!

Reply:
LGTM!